File: src/main/java/am/petstore/PetStore/user/jwt/InvalidJwtAuthenticationException.java
package am.petstore.PetStore.user.jwt;

import org.springframework.security.access.AuthorizationServiceException;

public class InvalidJwtAuthenticationException extends AuthorizationServiceException {
    /**
     *
     */
    private static final long serialVersionUID = -761503632186596342L;

    public InvalidJwtAuthenticationException(String e) {
        super(e);
    }
}

File: src/main/java/am/petstore/petstore/user/jwt/InvalidJwtAuthenticationException.kt
package am.petstore.petstore.user.jwt

import org.springframework.security.access.AuthorizationServiceException

class InvalidJwtAuthenticationException(e: String?) : AuthorizationServiceException(e) {
    companion object {
        /**
         *
         */
        private const val serialVersionUID = -761503632186596342L
    }
}